showpass/Showpass-E2E-tests
# TA-70 Add tags to make the execution more flexible and custom.

**Description:**
From now on, each spec can be labeled with a tag and then executed as part of a test group. These tests can be tagged as ‘e2e’, ‘smoke’, ‘system’, ‘sanity’ etc.
Once labeled, only the corresponding scripts will get executed.
See below for details:
[npm: @cypress/grep](https://www.npmjs.com/package/@cypress/grep#test-suites) 

NOTE:
Currently all specs are marked as 'smoke' tests.

Example: 
npx cypress run --env grepTags=smoke

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-70

**How to test:**
Add various tags to the test scripts and run the run them. Ensure that the right specs get executed

**Test execution results:**

![Uploading TA-70-all-tests-passed.png…]()
